Firestore/Example/Tests/API/FSTAPIHelpers.m

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright 2017 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "Firestore/Example/Tests/API/FSTAPIHelpers.h"
+#import <Firestore/Source/API/FIRDocumentSnapshot+Internal.h>
+#import <Firestore/Source/API/FIRQuerySnapshot+Internal.h>
+#import <Firestore/Source/API/FIRSnapshotMetadata+Internal.h>
+#import <Firestore/Source/Core/FSTViewSnapshot.h>
+#import "Firestore/Example/Tests/Util/FSTHelpers.h"
+
+#import "FirebaseFirestore/FIRCollectionReference.h"
+#import "FirebaseFirestore/FIRDocumentReference.h"
+#import "FirebaseFirestore/FIRDocumentSnapshot.h"
+#import "FirebaseFirestore/FIRFieldPath.h"
+#import "FirebaseFirestore/FIRFirestore.h"
+#import "FirebaseFirestore/FIRGeoPoint.h"
+#import "FirebaseFirestore/FIRQuerySnapshot.h"
+#import "FirebaseFirestore/FIRSnapshotMetadata.h"
+#import "Firestore/Source/API/FIRCollectionReference+Internal.h"
+#import "Firestore/Source/API/FIRDocumentReference+Internal.h"
+#import "Firestore/Source/API/FIRFieldPath+Internal.h"
+#import "Firestore/Source/API/FIRFirestore+Internal.h"
+#import "Firestore/Source/API/FSTUserDataConverter.h"
+#import "Firestore/Source/Core/FSTQuery.h"
+#import "Firestore/Source/Core/FSTSnapshotVersion.h"
+#import "Firestore/Source/Core/FSTTimestamp.h"
+#import "Firestore/Source/Core/FSTView.h"
+#import "Firestore/Source/Local/FSTLocalViewChanges.h"
+#import "Firestore/Source/Local/FSTQueryData.h"
+#import "Firestore/Source/Model/FSTDatabaseID.h"
+#import "Firestore/Source/Model/FSTDocument.h"
+#import "Firestore/Source/Model/FSTDocumentKey.h"
+#import "Firestore/Source/Model/FSTDocumentSet.h"
+#import "Firestore/Source/Model/FSTFieldValue.h"
+#import "Firestore/Source/Model/FSTMutation.h"
+#import "Firestore/Source/Model/FSTPath.h"
+#import "Firestore/Source/Remote/FSTRemoteEvent.h"
+#import "Firestore/Source/Remote/FSTWatchChange.h"
+#import "Firestore/Source/Util/FSTAssert.h"
```

Comment:
Most of these imports are unused.  Per AppCode, this is the set you need:
```
#import "Firestore/Example/Tests/API/FSTAPIHelpers.h"
#import <Firestore/Source/API/FIRDocumentSnapshot+Internal.h>
#import <Firestore/Source/API/FIRQuerySnapshot+Internal.h>
#import <Firestore/Source/API/FIRSnapshotMetadata+Internal.h>
#import <Firestore/Source/Core/FSTViewSnapshot.h>

#import "FirebaseFirestore/FIRCollectionReference.h"
#import "FirebaseFirestore/FIRDocumentReference.h"
#import "FirebaseFirestore/FIRDocumentSnapshot.h"
#import "FirebaseFirestore/FIRFirestore.h"
#import "FirebaseFirestore/FIRGeoPoint.h"
#import "FirebaseFirestore/FIRQuerySnapshot.h"
#import "FirebaseFirestore/FIRSnapshotMetadata.h"
#import "Firestore/Source/API/FIRCollectionReference+Internal.h"
#import "Firestore/Source/API/FIRFirestore+Internal.h"
#import "Firestore/Source/Core/FSTQuery.h"
#import "Firestore/Source/Model/FSTDocument.h"
#import "Firestore/Source/Model/FSTDocumentKey.h"
#import "Firestore/Source/Model/FSTDocumentSet.h"
#import "Firestore/Source/Model/FSTPath.h"
```

Reply:
done, also clean-up more and make style consistent (<> vs "")